.github/workflows/ci.yml

```
@@ -0,0 +1,421 @@
+        run: corepack enable || true
+
+      - name: Cache (yarn)
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/yarn
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: ${{ runner.os }}-yarn-
+
+      - name: Detect package manager
+        id: pm_lint
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:45: Double quote to prevent globbing and word splitting

(shellcheck)